docker-compose.yaml

```diff
@@ -164,9 +164,9 @@ services:
   cb-spider:
     image: cloudbaristaorg/cb-spider:0.11.4
     container_name: cb-spider
-    # build:
-    #   context: ../cb-spider
-    #   dockerfile: Dockerfile
+    build:
+      context: ../cb-spider
+      dockerfile: Dockerfile
```

Comment:
Sorry, I cannot accept this section either.

Setting debugging mode and build mode is not meant for upstream.

If we keep it as is, for instance, other users who do not have `../cb-spider` will encounter an error when running `docker compose up -d`.

Please revert this change and try not to add these kinds of personal changes to `docker-compose.yaml` in your PR. :)